indexer/pkg/utils/hex.go

```diff
@@ -0,0 +1,80 @@
+package utils
```

Comment:
Not sure why we are rewriting all these functions. All this functionality should already be available in the go-ethereum/common pkg.